omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -138,13 +145,14 @@
     # Prepare the build env, these dependencies are only needed to build and
     # install the core integrations.
     #
-    command "#{pip} download --dest #{build_deps_dir} hatchling==0.25.1"
-    command "#{pip} download --dest #{build_deps_dir} setuptools==40.9.0" # Version from ./setuptools3.rb
-    command "#{pip} install wheel==0.34.1"
-    command "#{pip} install pip-tools==6.4.0"
+    command "#{pip} download --dest #{build_deps_dir} hatchling==0.25.1", :env => pre_build_env, :log_level => "debug"
```

Comment:
It doesn't seem like the `:log_level => "debug"` parameter had an effect? Let's remove it then.

Reply:
Right, fixed in [98c297a](https://github.com/DataDog/datadog-agent/pull/14473/commits/98c297a956f8c0a8919621fa495d3bcc214107a2) !